python/lsst/ip/diffim/imageDecorrelation.py

```diff
@@ -274,7 +274,7 @@ def run(self, scienceExposure, templateExposure, subtractedExposure, psfMatching
             corr = self.computeDiffimCorrection(kArr, varianceMean, targetVarianceMean)
 
         correctedImage = self.computeCorrectedImage(corr.corrft, subtractedExposure.image.array)
-        correctedPsf = self.computeCorrectedDiffimPsf(corr.corrft, psfImg.array)
+        # correctedPsf = self.computeCorrectedDiffimPsf(corr.corrft, psfImg.array)
```

Comment:
This, and the other commented line below, should either be deleted, or have a TODO attached to them with the ticket number where they will be restored (or deleted).